source/documentation/runbooks/010-disaster-recovery-plan.html.md.erb

```diff
@@ -0,0 +1,129 @@
+---
+owner_slack: "#data-catalogue"
+title: "Disaster recovery plan: Find MoJ data"
+last_reviewed_on: 2024-09-04
+review_in: 3 months
+---
+
+# <%= current_page.data.title %>
+
+## Overview
+This runbook provides step-by-step guidance for recovering the 'Find MoJ data' Django application hosted on the Ministry of Justice's internal Cloud Platform (CP) using AWS EKS.
+It covers the actions required by the Data Catalogue team in the event of a disaster, focusing on minimizing downtime and data loss.
+
+## Summary
+
+- Find MoJ data is a Django application that calls a backend service [DataHub] for all of its data. Both services are hosted on the MoJ's [Cloud Platform] (CP), and use CP for integrated services like the application database ([AWS RDS PostgreSQL]) and search service (AWS OpenSearch).
+
+- Both Find MoJ data and DataHub deployments have RDS PostgreSQL databases deployed via CP and so get [automated daily backups] and the last 7 are retained.
+
+  > **Note**: due to an implementation bug, as of 2024-09-03 the snapshot backups are taken _**every other morning**_, meaning the oldest backup is 12-13 days old, and that the most recent backup can be up to 48 hours old.
+
+- Following loss or corruption of database data, we can restore from one of these backups by [following CP's RDS snapshot restore guidance].
```

Comment:
Just checking - we can re-deploy a bare DataHub instance and re-run the metadata ingestion. Are we not mentioning this to future-proof against user-generated entries?

Reply:
That, and it retains any usage metadata and user information within DataHub and FMD.

Reply:
I've now mentioned that here, along with more explicit detail later